draft-ietf-quic-transport.md

```diff
@@ -1133,6 +1134,14 @@ truncate_connection_id (0x0004):
   length.  Omitting the parameter indicates that the endpoint relies on the
   connection ID being present in every packet.
 
+max_packet_size (0x0005):
+
+: The maximum packet size parameter places a limit on the size of packets that
+  the endpoint is willing to receive, encoded as an unsigned 16-bit integer.
+  The default for this parameter is the maximum permitted UDP payload of 65527.
+  Values below 256 are invalid.  The initial packet from a client is exempt from
```

Comment:
1280 not 256?  Also, do we want to mention that the peer is indicating it will drop packets larger than the limit?